src/solver/conv/conv_wino_rage_RxS.cpp

```diff
@@ -0,0 +1,282 @@
+                                                          const ProblemDescription& problem)
+{
+    std::ignore = ctx;
+    std::ignore = problem;
+    return -2.0f;
+}
+
+template <uint32_t Winodata, uint32_t Winofilter>
+ConvSolution
+ConvWinoRageRxSCommon<Winodata, Winofilter>::GetSolution(const ExecutionContext& ctx,
+                                                         const ProblemDescription& problem,
+                                                         bool fused,
+                                                         bool do_bias,
+                                                         miopenActivationMode_t activ_mode)
+{
+    // Kernel args
+
+    WinoShaderArgsV2 args;
+    if(!args.SetConvParams(problem))
+    {
+        MIOPEN_THROW(miopenStatusInternalError);
+    }
+    args.SetStrides(problem);
+    args.SetActivParams(activ_mode);
+
+    auto flags = WinoShaderFlagsV2::F_NKCHR_STRIDES | WinoShaderFlagsV2::F_TENSOR_OFFSETS |
+                 WinoShaderFlagsV2::F_USE_ACTIVATION_MODE | WinoShaderFlagsV2::F_DENORMS_RND_ENABLE |
+                 WinoShaderFlagsV2::F_USE_EXTENDED_FLAGS_64;
+    if(args.G != 1)
+        flags |= WinoShaderFlagsV2::F_GROUPED_CONVOLUTION;
+    if(problem.IsDirectionBackwardData())
+        flags |= WinoShaderFlagsV2::F_REVERSE_R | WinoShaderFlagsV2::F_REVERSE_S;
+    if(do_bias)
+        flags |= WinoShaderFlagsV2::F_BIAS;
+
+    auto nGroups = getNGroups(ctx);
+    args.SetShaderParams(nGroups, flags, 0, 0);
+
+    // Kernel name and file
+
+    std::string kernelVersion;
+    if(args.R <= 3 && args.S <= 3)
+    {
+        kernelVersion = "_v4_6_0";
+    }
+    else
+    {
+        kernelVersion = "_v4_7_0";
+    }
+    std::string kernelName = "miopenSp3AsmConvRage" + kernelVersion;
+    std::string kernelFile = "Conv_Winograd_Rage" + kernelVersion;
+
+    /// \todo Add case for gfx12 kernels
+    kernelName += "_gfx9";
+
+    std::string kernelPostfix;
+
+    if(problem.IsFp16())
+    {
+        kernelPostfix += "_fp16_fp32acc";
+    }
+    else
+    {
+        MIOPEN_THROW(miopenStatusInternalError);
+    }
+
+    if constexpr(Winodata == 2 && Winofilter == 3)
+    {
+        kernelPostfix += "_f2x3";
+    }
+    else
+    {
+        static_assert(Winodata == 2 && Winofilter == 3);
+    }
+
+    kernelPostfix += "_stride1";
+
+    kernelName += kernelPostfix;
+    kernelFile += kernelPostfix;
+    kernelFile += ".s";
+
+    // Kernel info
+
+    KernelInfo kernelInfo;
+
+    /// Kernel doesn't need ROCM_METADATA_VERSION, but AmdgcnAssemble()
+    /// uses it to find out required CO version (hack).
+    /// \todo Delete when COv2 support is removed.
+    KernelBuildParameters options{
+        {"ROCM_METADATA_VERSION", 5},
+    };
+    kernelInfo.comp_options = options.GenerateFor(kbp::GcnAsm{});
+    kernelInfo.comp_options += std::string(" -mcumode -mwavefrontsize64");
+
+    /// \todo Add case for gfx12 wgp size of 384U
+    uint64_t wgSize = 768U;
```

Comment:
Instead of hardcoding, consider querying the wavefront/group size from the device - or at least defining it as named constant. Something like this: `constexpr uint64_t DefaultGfx9WgSize = 768;`